website/source/docs/providers/google/r/storage_bucket.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "google"
+page_title: "Google: google_storage_bucket"
+sidebar_current: "docs-google-resource-storage"
+description: |-
+  Creates a new bucket in Google storage.
+---
+
+# google\_storage\_bucket
+
+Creates a new bucket in Google cloud storage service(GCS). Currently, it will not change location nor ACL once a bucket has been created with Terraform. For more information see [the official documentation](https://cloud.google.com/storage/docs/overview) and [API](https://cloud.google.com/storage/docs/json_api).
+
+
+## Example Usage
+
+Example creating a private bucket in standard storage, in the EU region. 
+
+```
+resource "google_storage_bucket" "image-store" {
+	name = "image-store-bucket"
+	predefined_acl = "private"
+	location = "EU"
+}
+
+```
+
+* `name` - (Required) The name of the bucket.
+* `predefined_acl` - (Optional) The [canned GCS ACL](https://cloud.google.com/storage/docs/access-control#predefined-acl) to apply. Defaults to "private".
+  * `authenticatedread`
```

Comment:
Sorry I meant "authenticatedread" is specified in the Google documentation as either authenticated-read or authenticatedRead.  It looks like it works ok for you using the lowercase form, probably because some code is doing a case-insensitive check.  But I don't think it's officially supported so we probably shouldn't advertise it.